.gitignore

```diff
@@ -44,3 +44,5 @@ apps/forms-flow-ai/forms-flow-web/node_modules
 apps/forms-flow-ai/forms-flow-web/package-lock.json
 request-management-api/request_api/__pycache__/__init__.cpython-38.pyc
 request-management-api/request_api/__pycache__/__init__.cpython-38.pyc
+
+request-management-api/src
```

Comment:
Do we want this?  Shouldn't we normally commit our source folder to git?

Reply:
you have removed this folder for dev when we faced the deployment issue last day with sbc-common-components folder. But this should be added in .gitignore, because it might get created again in our local when we run it.